solidjs-tailwind/src/routes.js

```diff
@@ -0,0 +1,6 @@
+const ROUTES = {
+  HOME: '/',
```

Comment:
we can make this like so also in other to distinguish auth required pages from others
```js
HOME: {
path: '/',
requiresAuth: true,
}
```

Reply:
My idea was to split pages and routes. The page knows whether it needs authentication or not, the route is just the pointer to that page.
Maybe `PATHS` is a better name for this constant?